File: tests/fixtures/multi-lang/greeter.ts
// Minimal TypeScript fixture for the install-matrix smoke test.
// Keep small — the smoke gate only needs `codehub analyze` + `codehub query`
// to find at least one `export default` hit somewhere in the fixture.

export interface Greeting {
  language: string;
  text: string;
}

export default function greet(name: string): Greeting {
  return { language: "en", text: `Hello, ${name}!` };
}

File: tests/fixtures/multi-lang/greeter.go
// Minimal Go fixture for the install-matrix smoke test.
package multilang

// Greeting is a localized greeting.
type Greeting struct {
	Language string
	Text     string
}

// Greet returns a greeting for the supplied name.
func Greet(name string) Greeting {
	return Greeting{Language: "en", Text: "Hello, " + name + "!"}
}
